lib/internal/test_runner/harness.js

```diff
@@ -105,7 +103,7 @@ function collectCoverage(rootTest, coverage) {
 }
 
 function setup(root) {
-  if (wasRootSetup.has(root)) {
+  if (root.startTime !== null) {
```

Comment:
Doesn’t this change mean that someone could create a Test themselves, and be able to monkey with the harness? By using a weakest you’re ensuring that only things you create are treated as such.

Reply:
I don't think userland has access to any of the necessary APIs (the `Test` class or any of the harness internals) to do that.